bulk/eng_bulk_load-AssetRecord.csv

```diff
@@ -181,9 +177,9 @@ CGCON-GLDRCP-00387,OL000356,Sensor,1,Mobile Asset Controller for Coastal Glider,
 CGCON-GLDRCP-00388,OL000357,Sensor,1,Mobile Asset Controller for Coastal Glider,GL388 Controller,Teledyne Webb,G2 Slocum Gliders,388,,,,,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00389,OL000358,Sensor,1,Mobile Asset Controller for Coastal Glider,GL389 Controller,Teledyne Webb,G2 Slocum Gliders,389,,,,,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00514,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL514 Controller,Teledyne Webb,G2 Slocum Gliders,514,,,,,Woods Hole Oceanographic Institution
+CGCON-GLDRCP-00559,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL559 Controller,Teledyne Webb,G2 Slocum Gliders,559,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
```

Comment:
Looks good. If we want to be consistent we should add the comment "Was Global; now Coastal" for 514 and 564 as well